pipeline/experiment.py

```diff
@@ -150,7 +156,8 @@ class TrialNoteType(dj.Lookup):
     definition = """
     trial_note_type : varchar(24)
     """
-    contents = zip(('autolearn', 'protocol #', 'bad', 'bitcode', 'autowater', 'random_seed_start'))
+    contents = zip(('autolearn', 'protocol #', 'bad',
+                    'bitcode', 'autowater', 'random_seed_start'))
```

Comment:
similar to the 'mtl-' comment, might make sense to document which values go with which experiment in comments

Reply:
technically all of these note types are applicable to all 3 types of experiment

Reply:
thanks, yes wasn't sure here. thanks for clarifying, misread the reformat as addition. in that case default is good. :+1: